web/src/navbar_alerts.ts

```diff
@@ -188,6 +200,13 @@ export function initialize(): void {
             data_process: "notifications",
             rendered_alert_content_html: render_desktop_notifications_alert_content(),
         });
+    } else if (should_offer_to_update_timezone()) {
+        open({
+            data_process: "time_zone_update_offer",
+            rendered_alert_content_html: render_time_zone_update_offer_content({
+                browser_time_zone,
+            }),
+        });
```

Comment:
I am unsure of the priority of this alert, so I've put it here for now.

Reply:
Probably above `server_needs_upgrade` would be good?